docs/table/index.rst

```diff
@@ -228,7 +228,7 @@ Adding a new row of data to the table is as follows.  Note that the unit
 value is given in ``cm / s`` but will be added to the table as ``0.1 m / s`` in
 accord with the existing unit.
 
-  >>> t.add_row([-8, -9, 10 * u.cm / u.s, 11])
+  >>> t.add_row([-8, 'string', 10 * u.cm / u.s, 10])
```

Comment:
I think this was a bad behavior previously, where it accepted an int `-9` to a string column and silently converted.  No longer, inserting the wrong type into a string column raises an exception now.

Reply:
That seems better indeed. Probably best to add a test for it...